### DISCUSSION TITLE
Selective repo updates based on identifier

### DISCUSSION THREAD

**Discussion by jokay** [renovatebot/renovate#14927] [Request Help] [answered]
Is there a way to limit the updates to some repos by using some kind of identifier (environment variable)?

We would like to update 1-2 repositories on request without triggering the update to the other repositories.

The scheduled pipeline should still update all repositories, as we have it currently setup.

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter ?

**Reply by jokay** [reply]
Thx, this will work.
